test/widget_test.dart

```diff
@@ -4,27 +4,28 @@
 // utility that Flutter provides. For example, you can send tap and scroll
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
-
-import 'package:flutter/material.dart';
-import 'package:flutter_test/flutter_test.dart';
-
-import 'package:nativemoon/main.dart';
-
-void main() {
-  testWidgets('Counter increments smoke test', (WidgetTester tester) async {
-    // Build our app and trigger a frame.
-    await tester.pumpWidget(MyApp());
-
-    // Verify that our counter starts at 0.
-    expect(find.text('0'), findsOneWidget);
-    expect(find.text('1'), findsNothing);
-
-    // Tap the '+' icon and trigger a frame.
-    await tester.tap(find.byIcon(Icons.add));
-    await tester.pump();
-
-    // Verify that our counter has incremented.
-    expect(find.text('0'), findsNothing);
-    expect(find.text('1'), findsOneWidget);
-  });
-}
+//
+//import 'package:flutter/material.dart';
+//import 'package:flutter_test/flutter_test.dart';
+//
+//import 'package:nativemoon/main.dart';
+//import 'package:nativemoon/routes.dart';
+//
+//void main() {
```

Comment:
You should remove commented code. Why is this commented?

Reply:
This file is for testing purposes created in the beginning by flutter itself. When I changed the main only to create a Routes instance it started to throw errors. I decided to comment it for now.

Reply:
Please try to add a test for this component. If not don't leave a file full of comment with no use.

Reply:
Fixed.